master/docs/manual/customization.rst

```diff
@@ -932,18 +932,72 @@ For more information on the available commands, see :doc:`../developer/master-wo
     Step Progress
     BuildStepFailed
 
-Writing New Status Plugins
---------------------------
+Writing Dashboards with Flask
```

Comment:
Interesting about Flask instead of something like Klein. I do get why; Flask is vastly more popular and better understood.